radish/extensions/console_writer.py

```diff
@@ -45,6 +46,8 @@ def __init__(self):
         self.last_precondition = None
         self.last_background = None
 
+        self._placeholder_regex = re.compile(r'(<\w+>)', flags=re.UNICODE)
```

Comment:
You can actually use more characters then only the ones included in `\w`, like a hyphen.